tests/test_outlier.py

```diff
@@ -109,6 +109,19 @@ def make_data(
 data = make_data()
 
 
+def test_normalized_entropy():
```

Comment:
I believe this function will belong better in `test_util.py`. 

As for the failing tests, a nice way to ensure it does not throw an error would be to use `val = getattr(info, "attribute", None)` to check if each attribute exists, and skip the test if `val is None`?

Reply:
I moved the test as requested. `getattr` is indeed a good workaround. Instead of skipping `None`, I provide a reasonable fallback right-away, keeping the code simpler.